fiberplane/fiberplane
# Add ollama support

A cursory test of the Llama 3.1 model shows that it has become quite capable of supporting some of our request generation use cases. This makes for a good case to add support for using the Fiberplane Studio with Ollama.


To test:

- Set up ollama locally, make sure you have llama 3.1 (8B) running: `ollama run llama3.1`
- Rebuild types (`pnpm build:types` from the root)
- Go to settings, and set your model provider to `ollama`
